### PR TITLE
[ISSUE #3389] Replace Charset.forName() call with StandardCharsets.UTF_8

### LINKED ISSUE #3389: [Enhancement] Replace Charset.forName() call with StandardCharsets.UTF_8
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/constants/EventMeshConstants.java line 34

### Describe the solution you'd like

image
or
image
EventMeshConstants.DEFAULT_CHARSET -> EventMeshConstants.DEFAULT_CHARSET.name()
image
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by nikam14**
Fixes #3389 .

### Modifications

imported the StandardCharsets class and 
changed to StandardCharsets.UTF_8

### Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by nikam14** [comment]
@Alonexc can you help the test cases are failing

**Review by mxsm — COMMENTED** [review]
@nikam14  you can remove `public static final String DEFAULT_CHARSET = "UTF-8";` . And then replace it with Constants.DEFAULT_CHARSET

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/constants/EventMeshConstants.java [hunk 1]
```diff
@@ -31,7 +33,7 @@ public class EventMeshConstants {
 
     public static final String PROTOCOL_GRPC = "grpc";
 
-    public static final String DEFAULT_CHARSET = "UTF-8";
+    public static final String DEFAULT_CHARSET = StandardCharsets.UTF_8;
```
Comment (Alonexc) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/constants/EventMeshConstants.java:36]: public static final String DEFAULT_CHARSET = StandardCharsets.UTF_8.name();
or delete:
image

Then replace it with `Constants.DEFAULT_CHARSET.name()`:

In these classes need to be replaced:
image

**Comment by nikam14** [comment]
@Alonexc @mxsm  thanks your guidance.

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**